.github/workflows/rebuild-trivy-bundle.yaml

```
@@ -95,40 +90,39 @@ jobs:
 
   rebuild-trivy-bundles-images:
     needs: [obtain-trivy-image-tags, determine-if-workflow-should-run]
+    if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
     runs-on: ubuntu-latest
+    outputs:
+      UPDATED_TIME_STAMP: ${{ steps.rebuild_and_push_images.outputs.UPDATED_TIME_STAMP }}
```

Here and in other places too:
```suggestion
      UPDATED_TIMESTAMP: ${{ steps.rebuild_and_push_images.outputs.UPDATED_TIMESTAMP }}
```